### DISCUSSION TITLE
[Feature] Android: Remind the user of battery optimization

### DISCUSSION THREAD

**Discussion by etnoy** [immich-app/immich#2623] [Feature Request]
### The feature

We currently prompt the user to disable battery optimization on Android the first time the background upload service is enabled. However, this is easy to dismiss and then forget about since a new user is likely eager to try all features (happened to me!)

If we could get a periodic reminder about disabling battery optimization (like once every day or similar) that could help users get their assets uploaded. On my Galaxy S21 FE, not disabling battery optimization means the background service isn't able to upload large videos, making the whole upload stop.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by alextran1502** [comment]
I think there is a modal pop-up when we toggle the background upload button, isn't there?

**Reply by etnoy** [reply]
Yes, there's a modal that pops up once. However, as I stated above, someone stupid (like myself) will likely dismiss that dialog and then be endlessly frustrated because background sync never works. Some persistent information, say in the backup screen on the app, could have a text that says "battery optimization enabled, background sync might not function".
